mephisto/abstractions/blueprints/parlai_chat/parlai_chat_task_runner.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, *args, **kwargs):
 if TYPE_CHECKING:
     from mephisto.data_model.task_run import TaskRun
     from mephisto.abstractions.blueprint import AgentState
-    from mephsito.data_model.assignment import Assignment
+    from mephisto.data_model.assignment import Assignment
```

Comment:
oh wow, this typo is quite common: https://github.com/facebookresearch/Mephisto/search?q=mephsito

Reply:
All in my type checking code, which I haven't run in a while. I should add mypy signal!